lib/k8s/operation.ex

```diff
@@ -215,6 +216,10 @@ defmodule K8s.Operation do
   def put_label_selector(%Operation{} = op, %Selector{} = selector),
     do: put_query_param(op, @label_selector, selector)
 
+  @spec put_field_selector(Operation.t(), Selector.t()) :: Operation.t()
```

Comment:
Wouldn't it mke sense to (also) pdefine a `put_selector/N` function? I mean... the type is already defined by the passed Selector struct, no?